examples/simple-service-poc/simple_service.py

```diff
@@ -0,0 +1,181 @@
+        min_storage_gib=2.0,
+    )
+
+    async def start(self):
+        self._ctx.run("/golem/run/simulate_observations_ctl.py", "--start")
+        yield self._ctx.commit()
+
+    async def run(self):
+        while True:
+            await asyncio.sleep(10)
+            self._ctx.run(self.SIMPLE_SERVICE, "--stats")  # idx 0
+            self._ctx.run(self.SIMPLE_SERVICE, "--plot", "dist")  # idx 1
+
+            future_results = yield self._ctx.commit()
+            results = await future_results
+            stats = results[0].stdout.strip()
+            plot = results[1].stdout.strip().strip('"')
+
+            print(f"{TEXT_COLOR_CYAN}stats: {stats}{TEXT_COLOR_DEFAULT}")
+
+            plot_filename = (
+                    "".join(random.choice(string.ascii_letters) for _ in
+                            range(10)) + ".png"
+            )
+            print(f"{TEXT_COLOR_CYAN}downloading plot: {plot} to {plot_filename}{TEXT_COLOR_DEFAULT}")
+            self._ctx.download_file(plot, str(pathlib.Path(__file__).resolve().parent / plot_filename))
+
+            steps = self._ctx.commit()
+            yield steps
+
+    async def shutdown(self):
+        self._ctx.run("/golem/run/simulate_observations_ctl.py", "--stop")
+        yield self._ctx.commit()
+
+
+async def main(subnet_tag, driver=None, network=None):
+    async with Golem(
+        budget=1.0,
+        subnet_tag=subnet_tag,
+        driver=driver,
+        network=network,
+        event_consumer=log_summary(log_event_repr),
+    ) as golem:
+
+        print(
+            f"yapapi version: {TEXT_COLOR_YELLOW}{yapapi_version}{TEXT_COLOR_DEFAULT}\n"
+            f"Using subnet: {TEXT_COLOR_YELLOW}{subnet_tag}{TEXT_COLOR_DEFAULT}, "
+            f"payment driver: {TEXT_COLOR_YELLOW}{golem.driver}{TEXT_COLOR_DEFAULT}, "
+            f"and network: {TEXT_COLOR_YELLOW}{golem.network}{TEXT_COLOR_DEFAULT}\n"
+        )
+
+        start_time = datetime.now()
+
+        print(f"{TEXT_COLOR_YELLOW}starting {pluralize(NUM_INSTANCES, 'instance')}{TEXT_COLOR_DEFAULT}")
+
+        cluster = await golem.run_service(
+            SimpleService,
+            num_instances=NUM_INSTANCES,
+            expiration=datetime.now(timezone.utc) + timedelta(minutes=15))
+
+        def instances():
+            return [(s.provider_name, s.state.value) for s in cluster.instances]
+
+        def still_running():
+            return any([s for s in cluster.instances if s.is_available])
+
+        while datetime.now() < start_time + timedelta(minutes=2):
+            print(f"instances: {instances()}")
+            await asyncio.sleep(5)
+
+        print(f"{TEXT_COLOR_YELLOW}stopping instances{TEXT_COLOR_DEFAULT}")
+        cluster.stop()
+
+        cnt = 0
+        while cnt < 10 and still_running():
+            print(f"instances: {instances()}")
+            await asyncio.sleep(5)
+
+    print(f"instances: {instances()}")
+
+
+if __name__ == "__main__":
+    parser = build_parser("Test http")
```

Comment:
Is "Test http" an accurate description?

Reply:
haha, no :)

Reply:
done in https://github.com/golemfactory/yapapi/pull/303/commits/57f2c4882b078a0dbed67da6a9c856996deb2957